jepostule/pipeline/ami.py

```diff
@@ -6,6 +6,7 @@
 from jepostule.crypto import decrypt
 
 AMI_API_TIMEOUT_SECONDS = 10
+# FIXME make it dynamic prod / recette
 AMI_API_JOB_APPLICATION_ENDPOINT_URL = 'https://api-r.es-qvr.fr/partenaire/candidaturespontanee/v1/candidaturespontanee'
```

Comment:
should be prod URL if and only if client_id == lbb *and* this is production JP and not localdev JP (need to use setting)